content/en/docs/components/hyperparameter-tuning/resume-experiment.md

```diff
@@ -5,8 +5,9 @@ weight = 40
                     
 +++
 
-This page describes in detail how to modify running experiment and restart succeeded experiment.
-Follow this guide to known more about changing experiment execution process and use various
+This page describes in detail how to modify running experiment
+and restart succeeded experiment. Follow this guide to know more
+about changing the experiment execution process and use various
 resume policies for the Katib experiment.
 
 For details of how to configure and run your experiment, see the guide to
```

Comment:
- Grammar
- a11y

```suggestion
For details on how to configure and run your experiment, check the guide on
```

Reply:
Also, check the links please (e.g. `/docs/components/hyperparameter-tuning/experiment/` since some may be moving to `.../kativ/`.. I think?)